source/_includes/asides/component_navigation.html

```diff
@@ -37,6 +37,14 @@
     {%- endif -%}.
   </div>
 
+  {% if page.works_with %}
+  <div>
+    {%- for type in page.works_with -%}
+      <img src="../../images/works_with/works-with-{{ type }}.png" alt="Works via {{ type }}" class="works-with-badge">
```

Comment:
Use one of the margin helpers to add margin

Reply:
Alt text should include full text of the badge.

Reply:
Clicking on it should link to a page where a user can learn more what this means (the intro blog post)